### PR TITLE
Remove testnet-verifier

### PR DESCRIPTION
**Pull request by gbouv**

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 * A new config flag `launchAdditionalService` has been added. True by default, it controls the launch of services
 additional service like Forkmon, Prometheus and Grafana, as well as the transaction spammer. Is set to false, only the
 Ethereum network (EL _and_ CL nodes) are launched
+* The testnet-verifier has been removed, and with it its associated config params: `waitForFinalization`,
```
Comment (h4ck3rk3y) [inline docs/changelog.md:8]: ChangeLog entries should begin with verbs https://github.com/kurtosis-tech/kurtosis-internal-docs/blob/5382c9d560f43ce6666c30f8aba47eef79f25969/engineering-processes.md#the-changelog

Can we add a `Removals` Section too?

**Review by h4ck3rk3y — APPROVED** [review]

**Comment by gbouv** [comment]
@parithosh Kevin opened my eyes on what the testnet-verifier was, and I think we don't need it anymore right? I am tentatively removing it alongside its associated params. Let me know what you thing. If that still makes sense to keep it around, happy to close it

File: docs/changelog.md (hunk 1 above)
Comment (gbouv) [reply docs/changelog.md]: 👍 updated it, thanks for flagging

**Comment by parithosh** [comment]
Ah i'd like to keep the testnet-verifier actually, its useful to assert when a testnet is successful with quite a few granular parameters.

**Comment by gbouv** [comment]
Sounds good we can keep it! Thanks for checking me on this!

**Pull request closed without merging**